Nardri/trisixty-buys-API
# #162530905 User login

#### What does this PR do?
This PR creates a user login resource.

#### Description of Task to be completed?
- create a user login resource
- write tests for the resource

#### How should this be manually tested?
- Get the branch with `git fetch origin ft-user-login-162530905`
- Checkout to the branch with `git checkout  ft-user-login-162530905`
- Activate the env with `pipenv shell` and install dependencies with `pipenv sync`
- Run test `pytest`
- Run the application with `flask run`
- Make a POST request to`api/v1/user/login`

```
{
    "email": "megan@gmail.com",
    "password": "123456789"
}
```

Response data should have the following structure
```
{
    "data ": {
        "token": < JWT token >
     }  
    "message" : "User successfully logged in",
    "status" : "success"
}
```
#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162530905](https://www.pivotaltracker.com/story/show/162530905)

#### Screenshots (if appropriate)

#### Questions:
N/A
